src/pkg/inspection/controller.go

```diff
@@ -331,6 +332,20 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 		}
 	}
 
+	// Read config from InspectionPolicy, send assessment reports to Governor api if governor enabled.
+	if policy.Spec.Inspection.Assessment.Governor.Enabled {
+		governorConfig := policy.Spec.Inspection.Assessment.Governor
+		exporter := governor.GovernorExporter{
```

Comment:
Check whether  ApiUrl, ClusterId & ApiToken are not empty. if any one of them is empty, Throw the error.